content/understanding-urbit/history-people.md

```diff
@@ -0,0 +1,125 @@
+
+For those interested in contributing there are some simple instructions on [GitHub](https://github.com/urbit/urbit/blob/master/CONTRIBUTING.md#contributing-to-urbit), but we also run a [grants program](https://grants.urbit.org/) where we reward contributors with Urbit ID address space. Grants aren’t solely for pre-determined projects, either. If you have something you’d like to propose, we’d love to hear from you.
+
+We also run a bi-monthly course called ‘Hoon School’ to teach people about the system. Hoon School is designed for anyone with the most basic technical skills — you don’t have to be an expert programmer. If you’re interested, you should [apply](community/hoonschool). We accept everyone we can and only close the class when it gets too big.
+
+![](https://media.urbit.org/site/understanding-urbit/your-last-computer/your-last-computer-waves%402x.png)
+
+At the beginning, Urbit was just a few people with the right combination of imagination and discipline to try to rebuild computing.
+
+2002
+Urbit starts as an open-ended personal project. An “independent study PhD” to reinvent computing for a network-centric world.
+
+2008
+Nock, the foundation of Urbit, works. Coming in at 32 lines of code, that’s 1 line of code every two months. 
+
+2012
+Hoon, Urbit’s programming language, compiles itself to Nock. Writing Hoon is much easier than writing Nock.
+
+2013
+Arvo, Urbit’s OS kernel, boots and the first live Urbit network is started with a command-line chat.
+
+2014
+Tlon is founded to help support Urbit development (and is < 8 people for the next four years).
+
+2015
+Urbit has its first web interface and serves its own website.
+
+2016
+The first sale of Urbit address space sells out in four hours.
+
+2017
+Our test network runs for ten months without a reboot.
+
+2017
+Our private sale, shared only with our mailing list, sells out in six hours — limit two per person.
+
+2018
+Tlon sells about 8% of its stake in the network to accelerate Urbit development.
+urbit.live gets going and starts selling Urbit IDs to the public.
+
+2019
+Tlon spends the year stabilizing Arvo and building OS 1.
+ 
+![](https://media.urbit.org/site/understanding-urbit/your-last-computer/your-last-computer-waves%402x.png)
+
+Building Urbit is about building a future where technology doesn’t control us. Instead, we look forward to a future where technology is simpler, more reliable and less invasive. 
+
+Keep in touch while we make build this vision into reality by [joining our mailing list](https://urbit.us11.list-manage.com/subscribe/post?u=972a03db9e0c6c25bb58de8c8&id=be143888d2) or following us on [twitter/urbit](https://twitter.com/urbit) or [github/urbit](https://github.com/urbit). 
```

Comment:
I'm unsure if "twitter/urbit" is the intentional link name.

Reply:
let's just call it "twitter"